internal/Picker/Picker.js

```diff
@@ -406,6 +526,7 @@ const PickerBase = kind({
 const PickerDecorator = compose(
 	IdProvider({generateProp: null}),
 	Changeable,
+	Changeable({prop: 'reverseTransition'}),
```

Comment:
I agree with this PR overall except data of `onChange` to an app. In this change, `onChange` brings `reverseTransition` value also to an app and it is not desired behavior. `reverseTransition` is used basically for `ViewManager` and not meaningful data to apps at all.

It looks better to filter the unexpected property between two `Changeable` HoCs.